main/bind/named.ca

```diff
@@ -1,85 +0,0 @@
-;       This file holds the information on root name servers needed to
-;       initialize cache of Internet domain name servers
-;       (e.g. reference this file in the "cache  .  <file>"
-;       configuration file of BIND domain name servers).
-;
-;       This file is made available by InterNIC 
-;       under anonymous FTP as
-;           file                /domain/named.root
-;           on server           FTP.INTERNIC.NET
-;       -OR-                    RS.INTERNIC.NET
-;
-;       last update:    Feb 04, 2008
-;       related version of root zone:   2008020400
```

Comment:
So this very old version calls itself `named.root` and was just rotting away in our git repository.